splitio/client/client.go

```diff
@@ -40,7 +40,7 @@ type SplitClient struct {
 	validator           inputValidation
 	factory             *SplitFactory
 	impressionListener  *impressionlistener.WrapperImpressionListener
-	impressionManager   provisional.ImpressionManager
+	impressionManager   *provisional.ImpressionManagerImpl
```

Comment:
why not interface?

Reply:
it is to avoid making a  breaking change in commons